ChangeLog.d/issue4084.txt

```diff
@@ -0,0 +1,27 @@
+Removals
+    * Remove MD2, MD4, RC4, Blowfish and XTEA cryptographic primitives and
+      ciphersuites. Remove the following library files: arc4.c/h, blowfish.c/h,
+      md2.c/h, md4.c/h, xtea.c/h and belonging csr/crt/pem files. Remove the
```

Comment:
I don't think we need to list every symbol that was removed, it doesn't really bring that much more information. Suggestion: "Remove all support for MD2, MD4, RC4, Blowfish and XTEA. This removes the corresponding modules and all their APIs and related configuration options. Fixes #4084." Otherwise the ChangeLog will just be way too long for anyone to read...